src/EA.Weee.RequestHandlers/AatfReturn/ObligatedReceived/AddObligatedReceivedHandler.cs

```diff
@@ -24,18 +24,18 @@ public async Task<bool> HandleAsync(AddObligatedReceived message)
             authorization.EnsureCanAccessExternalArea();
 
             var aatfWeeReceived = new WeeeReceived(
-                await obligatedReceivedDataAccess.GetSchemeId(message.OrganisationId),
-                await obligatedReceivedDataAccess.GetAatfId(message.OrganisationId),
+                message.SchemeId,
+                message.AatfId,
                 message.ReturnId);
 
-            var aatfWeeReceivedAmount = new List<WeeeReceivedAmount>();
+            var aatfWeeeReceivedAmount = new List<WeeeReceivedAmount>();
 
             foreach (var categoryValue in message.CategoryValues)
             {
-                aatfWeeReceivedAmount.Add(new WeeeReceivedAmount(aatfWeeReceived, categoryValue.CategoryId, categoryValue.HouseholdTonnage, categoryValue.NonHouseholdTonnage));
+                aatfWeeeReceivedAmount.Add(new WeeeReceivedAmount(aatfWeeReceived, categoryValue.CategoryId, categoryValue.HouseholdTonnage, categoryValue.NonHouseholdTonnage));
             }
 
-            await obligatedReceivedDataAccess.Submit(aatfWeeReceivedAmount);
+            await obligatedReceivedDataAccess.Submit(aatfWeeeReceivedAmount);
```

Comment:
Can we update the HandleAsync_WithValidInput_SubmittedIsCalledCorrectly test to ensure that the submit is called with the WeeeReceived that matches with the AATF and Scheme Id